docs/API.md

```diff
@@ -2,46 +2,56 @@
 
 ### gulp.src(globs[, options])
 
-Emits files matching provided glob or an array of globs. 
-Returns a [stream](http://nodejs.org/api/stream.html) of [Vinyl files](https://github.com/wearefractal/vinyl-fs) 
-that can be [piped](http://nodejs.org/api/stream.html#stream_readable_pipe_destination_options) 
-to plugins.
+Emits files matching provided glob or an array of globs.
```

Comment:
I would get rid of the "an" here, it makes it more obvious that the files are matching the array of globs, not that an array of globs is emitted.